### DISCUSSION TITLE
Roadmap for wn and wn-editor

### DISCUSSION THREAD

**Discussion by francis-dion** [goodmami/wn#164] [Q&A] [answered]
Hi,
@goodmami, @fcbond and others, I'm curious about you're thoughts on the future wn.
I went through the Issues list, and I see good opportunities for improvements and extensions there, but I'm interested about the bigger picture of where you see the library going and evolving towards.

**Comment by goodmami** [comment, marked as answer]
Hi, development has slowed since I transitioned to my new job, but I'm still maintaining it and @fcbond plans to be more involved and maybe have a student contribute as well. We are also open to offers to help maintain the project.

While there are not currently many plans for new features (wn-editor is not maintained by us but it is pushing forward on the creation/editing of wordnets), there are a few areas where we plan to keep active:

1. Fixing bugs and, secondly, optimizations
2. Supporting the latest WN-LMF format (e.g., [version 1.2](https://github.com/globalwordnet/schemas/pull/54) when it is released)
3. Documentation (more guides, more complete coverage of the API)
4. Increasing/retaining users and potential contributors through outreach and responding to questions, etc.

We are happy to discuss new features and things you can do with Wn, but I'm more likely to suggest that it reside in a separate project rather than Wn (as was the case with wn-editor), especially if it comprises a lot of new code, basically because I don't want to take on additional responsibility for maintaining it. This may change if we have more active contributors who can help out.

@fcbond, anything to add?
